.env.example

```diff
@@ -4,23 +4,8 @@ GENERATE_SOURCEMAP=<TRUE | FALSE>
 # port to use for serving the application on (defaults to 3000)
 PORT=<port number>
 
+REACT_APP_NETWORK=<TESTNET | MAINNET>
 
 
-REACT_APP_NETWORK=<LOCAL | TESTNET | MAINNET>
```

Comment:
we don't need this anymore as `rpcs and lcds` are now defined in `constants/chains`

Reply:
Will make sure it's removed from AWS side.

Reply:
@ap-justin This is still needed locally for development? I get mainnet endowment data when I remove this from my local `.env`.